test/utils-tests.jl

```diff
@@ -144,6 +144,38 @@ let
     @test alt  ≈ 10
 end
 
+# Test eq2hor
+# The values used for the testset are from running the code. However they have been
+# correlated with the output from eq2hor routine of IDL AstroLib, with
+# differences only in the least significant digits.
+@testset "eq2hor" begin
+    alt_o, az_o, ha_o = eq2hor(259.20005705918317, 49.674706171288655, AstroLib.J2000,
```

Comment:
In addition, a good test is to check that `eq2hor` and `hor2eq` are actually one the inverse of the other, but this doesn't seem to be the case:
```julia
julia> hor2eq(eq2hor(45, 60, 2e6)[1:2]..., 2e6)
(101.57713976172258, 64.04620027754098, 14.761079887794974)
```

Instead with original AstroLib:
```
GDL> eq2hor, 45d0, 60d0, 2d6, alt, az
% EQ2HOR: Using latitude and longitude for Pine Bluff Observatory
GDL> hor2eq, alt, az, 2d6, ra, dec
GDL> print, ra, dec, format='(f25.17)'
     45.00017354284745608
     59.99999514335013373
```
Precision isn't very high, but you can see that `ra` and `eq` approximately match the coordinates passed to `eq2hor`.